src/_posts/languages/ruby/2000-01-01-start.md

```diff
@@ -50,10 +50,19 @@ automatically be set by the platform:
 
 ## Bundler Version
 
-Our deployment system is automatically using the latest available bundler version.
+The buildpack installs a version of Bundler based on the major and minor
+version listed in the `Gemfile.lock` file, under the `BUNDLED WITH` key:
 
-* Bundler 1: **1.17.3**
-* Bundler 2: **2.3.10**
+| `BUNDLED WITH` | Bundler Version | Req. Ruby Version |
+| -------------: | --------------: | ----------------: |
+| `2.6.x`        | `2.6.2`         | >= `3.1.0`        |
+| `2.5.x`        | `2.5.23`        | >= `3.0.0`        |
+| `2.4.x`        | `2.4.22`        | >= `2.6.0`        |
+| `2.3.x`        | `2.3.25`        | >= `2.3.0`        |
+| `2.2.x`        | `2.3.25`        | >= `2.3.0`        |
+| `2.1.x`        | `2.3.25`        | >= `2.3.0`        |
+| `2.0.x`        | `2.3.25`        | >= `2.3.0`        |
+| `1.x`          | `1.17.3`        | >= `1.8.7`        |
```

Comment:
question: do we really still support all these versions? Aren't there some of them that are deprecated and we could remove them from the table?

Reply:
Ok,
- I removed the EOL versions of Ruby from our doc ;
- I removed the deprecated versions of Bundler from our doc.

Reply:
Nice, thanks a lot!